src/main/java/org/folio/innreach/domain/entity/TransactionPatronHold.java

```diff
@@ -18,10 +18,10 @@
 public class TransactionPatronHold extends TransactionHold {
 
   @Column(name = "title")
-  private String title;
+  private String titlePatron;
 
```

Comment:
If you specify getter/setter for the field explicitly then there will be no need to update usage of it in other places. JPA (`InnReachTransactionSpecification` in particular) will use it as "titlePatron" (since it doesn't rely on getter/setter by default) while other classes can keep using it as title.

```
  @Getter(AccessLevel.NONE)
  @Setter(AccessLevel.NONE)
  @Column(name = "title")
  private String titlePatron;

  public String getTitle() {
    return titlePatron;
  }

  public void setTitle(String title) {
    this.titlePatron = title;
  }
```